tests/docker_das_alpine/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM docker:20.10.21-dind-alpine3.16
+RUN apk add python3 --update --no-cache python3 && ln -sf python3 /usr/bin/python
+RUN apk add git
+RUN apk add bash
+RUN apk add curl
+RUN python3 -m ensurepip
+WORKDIR /das
+RUN pip3 install --no-cache --upgrade pip setuptools
+RUN pip3 install hyperon-das
+RUN git clone https://github.com/singnet/das-toolbox.git
+RUN pip3 install -r das-toolbox/src/requirements.txt
+COPY config.json /root/.das/config.json
+COPY entrypoint.sh entrypoint.sh
+COPY animals.metta animals.metta
+EXPOSE 8080
+CMD ["/bin/bash","-c","./entrypoint.sh"]
```

Comment:
```dockerfile
FROM docker:20.10.21-dind-alpine3.16
RUN apk add --update --no-cache \
    python3 \
    git \
    bash \
    curl
RUN ln -sf </full/absolute/path/>python3 /usr/bin/python \
    && /usr/bin/python -m ensurepip
WORKDIR /das
RUN git clone https://github.com/singnet/das-toolbox.git
RUN pip3 install --no-cache --upgrade \
    pip \
    setuptools \
    hyperon-das \
    && pip3 install -r das-toolbox/src/requirements.txt
COPY config.json /root/.das/config.json
COPY entrypoint.sh entrypoint.sh
COPY animals.metta animals.metta
EXPOSE 8080
CMD ["/bin/bash","-c","./entrypoint.sh"]
```

Reply:
Thanks, I got 12s of improvement when building the image